app.js

```diff
@@ -42,6 +42,10 @@
         process.exit(0);
     }
 
+    // Disabling the "X-Powered-By" header to avoid fingerprinting.
+    // http://expressjs.com/en/advanced/best-practice-security.html#at-a-minimum-disable-x-powered-by-header
```

Comment:
Sure, my fault, I have to close this PR and open another to change the commit message.